test/data/Cargo3.toml

```diff
@@ -0,0 +1,120 @@
+[package]
+name = "ripgrep"
+version = "14.1.0"  #:version
```

Comment:
I think it's a good idea to have some test cases like this with trailing comments for fields!